eachdist.ini

```diff
@@ -41,17 +41,6 @@ packages=
     opentelemetry-test-utils
     tests
 
-[experimental]
-version=1.10a0
-
-packages=
-    opentelemetry-exporter-prometheus-remote-write
-    opentelemetry-api
-    opentelemetry-sdk
-    opentelemetry-exporter-otlp-proto-grpc
-    opentelemetry-exporter-otlp-proto-http
-    opentelemetry-exporter-otlp
```

Comment:
Is this metadata tracked somewhere else now?

Reply:
Do you think we need that anymore? The `prometheus-remote-write` will be restored to the contrib repo and the experimental stuff is kept under private paths in the rest of the packages so I thought of removing this completely.

Reply:
Makes sense, I wasn't really sure what this did but sounds like you have it figured out